aries-site/src/data/wcag/components.json

```diff
@@ -862,5 +862,519 @@
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.8",
+        "label": "Media Alternative (Prerecorded)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.2.9",
+        "label": "Audio-only (Live)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "conditional",
+        "notes": "Needs additional ARIA labelling to be added for screen reader users."
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.1",
+        "label": "Use of Color",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.2",
+        "label": "Audio Control",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.3",
+        "label": "Contrast (Minimum)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.4",
+        "label": "Resize Text",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.5",
+        "label": "Images of Text",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.6",
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "failed",
+        "notes": "Unable to access calendar when scrolling on small viewport"
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "failed",
+        "notes": ""
```

Comment:
Could we add some notes here? What happens when we try to adjust line-height, letter-spacing, and word-spacing that is causing this rule to fail?